src/dfx/src/lib/builders/mod.rs

```diff
@@ -174,78 +177,8 @@ pub trait CanisterBuilder {
                 )
             })?;
             eprintln!("  {}", &output_did_js_path.display());
-            // index.js
-            let mut language_bindings = crate::util::assets::language_bindings()
```

Comment:
Moved this to a separate function that takes a param "ts" or "js" and will only generate the files for the provided extension.